### PR TITLE
feat: store raw stopwords and rewrite them at export

### PR DESCRIPTION
**Pull request by perierc**
### What

Currently, the parser reads the raw stopwords from the input taxonomy file, normalizes them and stores only these normalized stopwords in the Neo4J database.

This PR makes the parser store the raw stopwords also, to be able to rewrite them at export.

**WARNING: This PR will make previously open projects impossible to export.**

### Part of 

See issue #357

### CODE REVIEW

File: parser/openfoodfacts_taxonomy_parser/unparser.py
Comment (perierc) [inline parser/openfoodfacts_taxonomy_parser/unparser.py:1]: Note: this breaks the export for projects parsed before this PR.

File: parser/openfoodfacts_taxonomy_parser/parser/taxonomy_parser.py [hunk 1]
```diff
@@ -243,15 +243,18 @@ def _harvest_entries(self, filename: str, entries_start_line: int) -> Iterator[N
                     id = "stopwords:" + str(index_stopwords)
                     data = self._set_data_id(data, id, line_number)
                     index_stopwords += 1
+                    line = line[10:]
```
Comment (alexgarel) [inline parser/openfoodfacts_taxonomy_parser/parser/taxonomy_parser.py:246]:
```suggestion
                    # remove "stopwords:" part
                    line = line[10:]
```

File: parser/openfoodfacts_taxonomy_parser/parser/taxonomy_parser.py [hunk 2]
```diff
@@ -243,15 +243,18 @@ def _harvest_entries(self, filename: str, entries_start_line: int) -> Iterator[N
...
+                    tags = [words.strip() for words in line[3:].split(",")]
```
Comment (alexgarel) [inline parser/openfoodfacts_taxonomy_parser/parser/taxonomy_parser.py:247]:
```suggestion
                    # compute raw values outside _get_lc_value as it removes stop words !
                    tags = [words.strip() for words in line[3:].split(",")]
```

**Review by alexgarel — APPROVED** [review]
Very good !

Just added two comment if you want to integrate them before merging.

**Pull request merged**